api/admin/controller.py

```diff
@@ -633,10 +634,9 @@ def circulation_events(self):
         return dict({ "circulation_events": events })
 
     def bulk_circulation_events(self):
-        annotator = AdminAnnotator(self.circulation)
-
-        start = flask.request.args.get("start")
-        end = flask.request.args.get("end")
+        default = str(datetime.today()).split(" ")[0]
+        date = flask.request.args.get("date", default)
```

Comment:
Is this a string or a date? If it's a string, how does line 659 work? If it's a date, why do you need to parse it on line 639?